drivers/cc110x/include/cc110x_params.h

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (C) 2017 Marian Buschsieweke <marian.buschsieweke@ovgu.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup   drivers_cc110x
+ * @{
+ *
+ * @file
+ * @brief     cc110x board specific configuration
+ *
+ * @author    Marian Buschsieweke <marian.buschsieweke@ovgu.de>
+ */
+
+#ifndef CC110X_PARAMS_H
+#define CC110X_PARAMS_H
+
+#include "board.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    Default parameters for the cc110x driver
+ *
+ * These values are based on the msbiot board
+ * @{
+ */
+#ifndef CC110X_SPI
+#define CC110X_SPI                 SPI_DEV(0)
+#endif
+
+#ifndef CC110X_CS
+#define CC110X_CS                  GPIO_PIN(PORT_B, 12)
+#endif
+
+#ifndef CC110X_GDO0
+#define CC110X_GDO0                GPIO_PIN(PORT_C,  4)
+#endif
+
+#ifndef CC110X_GDO1
+#define CC110X_GDO1                GPIO_PIN(PORT_A,  6)
+#endif
+
+#ifndef CC110X_GDO2
+#define CC110X_GDO2                GPIO_PIN(PORT_C,  5)
+#endif
+/** @} */
```

Comment:
Please also add a `CC110X_PARAMS` define, like it's done in at86rf2xx driver. This way it allows a developer to define a global params define for multiple devices. With your change, this is not possible.